torch/optim/sgd.py

```diff
@@ -23,6 +23,19 @@ class SGD(Optimizer):
         >>> optimizer.step()
 
     __ http://www.cs.toronto.edu/%7Ehinton/absps/momentum.pdf
+    
+    .. note::
+        The implementation of SGD with Momentum/Nesterov subtly differs from
+        Sutskever et. al. and implementations in some other frameworks.
+        Considering the specific case of Momentum, the update can be written as
+        v = rho * v + g,
```

Comment:
Can you add math symbols via `:math`, as done in https://github.com/pytorch/pytorch/blob/master/torch/_torch_docs.py#L457 ?